fedimint-server/src/consensus/engine.rs

```diff
@@ -650,7 +660,17 @@ impl ConsensusEngine {
         }
 
         self.process_consensus_item_with_db_transaction(&mut dbtx.to_ref_nc(), item.clone(), peer)
-            .await?;
+            .await
+            .inspect_err(|err| {
+                // Rejected items are very common, so only trace level
+                trace!(
+                    target: LOG_CONSENSUS,
+                    %peer,
+                    item = ?DebugConsensusItem(&item),
+                    err = %err.fmt_compact_anyhow(),
+                    "Rejected consensus item"
+                );
+            })?;
```

Comment:
nit: items get rejected for being redundant too, not sure calling them rejected gives everyone the right intuition what that means.

Reply:
Since it prints `err`, it shows right there that it was "Rejected", so should be clear. Plus it's a trace log.